roles/arrstack/templates/buildarr/instances/sonarr-instances.yml.j2

```
@@ -0,0 +1,176 @@
+            definitions:
+              "Release Sources (Streaming Service)":
+                trash_id: 1B018E0C53EC825085DD911102E2CA36
+                strict_negative_scores: false
+                include_preferred_when_renaming: true
+              "P2P Groups + Repack/Proper":
+                trash_id: 71899E6C303A07AF0E4746EFF9873532
+                strict_negative_scores: false
+                include_preferred_when_renaming: false  # Consistent with guide.
+              "Low Quality Groups":
+                trash_id: EBC725268D687D588A20CBC5F97E538B
+                strict_negative_scores: false
+                include_preferred_when_renaming: false  # Consistent with guide.
+              "Optionals":
+                trash_id: 76e060895c5b8a765c310933da0a5357
+                strict_negative_scores: false
+                include_preferred_when_renaming: false  # Consistent with guide.
+                filter:
+                  include:
+                    - fa47da3377076d82d07c4e95b3f13d07  # Prefer Dolby Vision (HDR)
+                    - bc7a6383cbe88c3ee2d6396e1aacc0b3  # Prefer HDR
+                    - ea83f4740cec4df8112f3d6dd7c82751  # Prefer Season Packs
+                    - 6f2aefa61342a63387f2a90489e90790  # Dislike renamed/retagged releases
+                    - 19cd5ecc0a24bf493a75e80a51974cdd  # Dislike retagged/obfuscated groups
+                    - 6a7b462c6caee4a991a9d8aa38ce2405  # Dislike release ending: en
+                    - 236a3626a07cacf5692c73cc947bc280  # Dislike release containing: 1-
+                    # - cec8880b847dd5d31d29167ee0112b57  # Ignore 720p/1080p HEVC re-encodes (Golden Rule)
+                    - 436f5a7d08fbf02ba25cb5e5dfe98e55  # Ignore Dolby Vision without HDR10 fallback
+                    - f3f0f3691c6a1988d4a02963e69d11f2  # Ignore The Group -SCENE
+                    - 5bc23c3a055a1a5d8bbe4fb49d80e0cb  # Ignore so-called scene releases
+                    - 538bad00ee6f8aced8e0db5218b8484c  # Ignore Bad Dual Audio Groups
+                    - 4861d8238f9234606df6721df6e27deb  # Ignore AV1
+        download_clients:
+          definitions:
+            "Transmission (4K Shows)":
+              type: "transmission"
+              enable: false
+              host: "transmission"
+              port: 9091
+              use_ssl: false
+              url_base: "/transmission/"
+              username: null
+              password: null
+              # category: "4k"
+              directory: "/data/torrents/shows/4k"
+              recent_priority: "last"
+              older_priority: "last"
+              add_paused: false
+
+    # Sonarr instance for grabbing anime shows.
+    sonarr-anime:
+      hostname: {{ hostname_sonarr | default('localhost') }}
+      api_key: "{{ arrstack_instance_apikeys['sonarr-anime'] if arrstack_instance_apikeys is defined and 'sonarr-anime' in arrstack_instance_apikeys else 'ansible_api_key_not_found' }}"
```

Putting in an incorrect value when something is not found is an anti-pattern I believe.

Instead, let Ansible raise an error.

```suggestion
      api_key: "{{ arrstack_instance_apikeys['sonarr-anime'] }}"
```